Makefile

```diff
@@ -1,11 +1,11 @@
 
 install:
-	virtualenv .
+	python2.7 -m virtualenv .
```

Comment:
right, -m has been, but the virtualenv module distribution was unclear, i think i'm getting errors partly because i was already in a virtualenv, just verified it works on 2.7.6